### DISCUSSION TITLE
[Feature Request]: 2nd Inverter

### DISCUSSION THREAD

**Discussion by abmerlin** [wlcrs/huawei_solar#299] [Setting up the integration] [answered]
### Describe your feature request

your plugin for Home Assistant and Huawei SUN2000 works very well. I can also see my SUN2000-10KTL-M1 and Powermeter. Unfortunately, I can't see my SUN2000-17KTL-M2 which is also connected to the dongle. What can I do so that the data is also read out?

### Proper usage

- [x] I confirm that this is not a bug report
- [X] I confirm that this feature request is within the stated scope of the integration

**Comment by jires** [comment, marked as answer]
If you have 1 dongle only and multiple inverters (cascaded scenario), you need to provide all modbus addresses of the inverters, when you setting up the integration, inside the "slave IDs". It has to bee comma separated,  Like "1, 2, 3"

If you have multiple dongles, just add new instance of integration with information (IP, port, etc.) from new/other dongle.
